vmdb/app/helpers/application_helper.rb

```diff
@@ -2680,14 +2680,15 @@ def placeholder_if_present(password)
 
   def render_listnav_filename
     if @lastaction == "show_list" && !session[:menu_click] &&
-       %w(ems_cloud ems_cluster ems_infra host miq_template offline orchestration_stack repository
+       %w(container_node container_service ems_container container_group ems_cloud ems_cluster
+          ems_infra host miq_template offline orchestration_stack repository
           resource_pool retired service storage templates vm).include?(@layout) && !@in_a_form
       "show_list"
     elsif @compare
       "compare_sections"
     elsif %w(offline retired templates vm vm_cloud vm_or_template).include?(@layout)
       "vm"
-    elsif %w(action availability_zone cim_base_storage_extent cloud_tenant condition ontainer_group
+    elsif %w(action availability_zone cim_base_storage_extent cloud_tenant condition container_group
```

Comment:
Next time, please do typo fixes like this as a separate PR.  It's a different type of change and much easier to merge/review.